app/controllers/checks_controller.rb

```diff
@@ -44,7 +44,8 @@
   results = check.results.order(:created_at.asc)
   log = prepare_logs(results, is_ok: true)
   fails_log = prepare_logs(results, is_ok: false)
-  json({ log: log, fails_log: fails_log })
+  issues_log = get_issues(results);
+  json({ log: log, fails_log: fails_log, issues_log: issues_log })
```

Comment:
you can remove useless code here
plz use only `log` var which has structure like 
```ruby
[
  { time: '12:30:21', issues: 'joined issues here', response_time: 456 },
  { time: '12:35:15', issues: 'joined issues here', response_time: 317 }
]
```
and make appropriate changes in js file